src/Service/MethodRegistry.php

```diff
@@ -70,7 +70,7 @@ public function getMethods(): array
             if (!$method instanceof MethodInterface) {
                 throw new UnexpectedValueException(sprintf(
                     'Given method "%s" does not implement %s',
-                    $method,
+                    get_class($method),
```

Comment:
This isn't _necessary_ anymore, since I had to revert the changes to what `ModelData::__toString()` returns - but it was still a really bad idea to assume that will return the class name. I still think _this_ change is objectively good and worthwhile.